onedal/decomposition/pca.py

```diff
@@ -128,15 +129,20 @@ def _create_model(self):
         return m
 
     def predict(self, X, queue=None):
-        policy = self._get_policy(queue, X)
+        # Not supported with spmd policy so BasePCA must be specified
+        policy = BasePCA._get_policy(BasePCA, queue, X)
         model = self._create_model()
         X_table = to_table(X, queue=queue)
         params = self._get_onedal_params(X_table, stage="predict")
 
-        result = self._get_backend(
-            "decomposition", "dim_reduction", "infer", policy, params, model, X_table
+        # Not supported with spmd policy so BasePCA must be specified
+        result = BasePCA._get_backend(
+            BasePCA, "decomposition", "dim_reduction", "infer", policy, params, model, X_table
         )
         return from_table(result.transformed_data)
+    
+    def transform(self, X, queue=None):
+        return self.predict(X, queue=queue)
```

Comment:
```suggestion
    transform = predict
```
unless you need to have something specific with ```IncrementalPCA.transform.__name__``` that we need to worry about

Reply:
good call